### PR TITLE
Enable version increment without file, just in runtime

### PR DESCRIPTION
**Pull request by jonascavalcanti**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -23,6 +25,13 @@ func main() {
 	for index, opt := range os.Args {
 
 		switch opt {
+		case "version", "--version":
+			if len(os.Args[index:]) <= 1 {
+				fmt.Println("You need to set the file path of application that contain the application version")
+				os.Exit(1)
+			} else {
+				options.Version = os.Args[index+1]
+			}
```
Comment (thiago-scherrer) [inline main.go:34]: Early Return here, this `else` can be removed

File: main.go [hunk 2]
```diff
@@ -64,14 +73,35 @@ func main() {
...
-	fmt.Println("New Version:", newVersion)
 
-	manipulator.WriteVersionOnFile(opts.Filepath, oldVersion, newVersion)
+	if opts.Filepath != "" {
+		manipulator.WriteVersionOnFile(opts.Filepath, oldVersion, newVersion)
+		fmt.Println("Application Version:", oldVersion)
+		fmt.Println("New Version:", newVersion)
+	} else if opts.Version != "" {
+		//os.Setenv("newVersionApp", newVersion)
+		//fmt.Println("newVersionApp:", os.Getenv("newVersionApp"))
```
Comment (thiago-scherrer) [inline main.go]: These comments can be removed, to make the code more readable.

Comment (jonascavalcanti) [reply main.go]: Great. I am going to do that!!

**Comment by thiago-scherrer** [comment]
Can the `xversioner-mac` be removed too?

File: main.go (hunk 1 above)
Comment (jonascavalcanti) [reply main.go:34]: @thiago-scherrer I need set the variable or not?

Comment (thiago-scherrer) [reply main.go:34]: Like this @jonascavalcantineto => 
```
			if len(os.Args[index:]) <= 1 {
				fmt.Println("You need to set the file path of application that contain the application version")
				os.Exit(1)
			} 
			
			options.Version = os.Args[index+1]
			
                       case "filepath", "-f": 
                       .....
```

**Pull request merged**